.github/workflows/upload.yaml

```diff
@@ -0,0 +1,87 @@
+name: Upload to website
+
+on:
+  push:
+    branches:
+      - main
+    # probably specify a dedicated directory for website content that needs to be uploaded to the bucket
```

Comment:
So what we'll want to do for this is actually to not have the upload be a separate workflow. It needs to run as a final step or job here: https://github.com/Opentrons/opentrons-python-packages/blob/main/.github/workflows/build-packages.yaml which is properly based on path changes and tags.

Reply:
If you're comfortable with me doing so, I can push the changes to this branch @OtuyaC

Reply:
Sure you can combine it and tweak it as much as you are comfortable with. The end goal is to get it to the desired point right?

Reply:
I pushed a lil update to the infra-fixup-workflow branch you created, which I think can be merged instead of this branch. If that's fine with you we can actually close this PR.